lib/listener/helpers.js

```diff
@@ -21,6 +22,9 @@ module.exports = function () {
   const runAsyncHelpersHook = (hook, param, force) => {
     if (store.dryRun) return;
     Object.keys(helpers).forEach((key) => {
+      if (helpers[key].feature && param.file) {
```

Comment:
Please remove this part, then I'm merging

Reply:
Sure @DavertMik! Will make the change.
However, I still believe that having this part would really add value, if it is provided out of the box by CodeceptJS.

Reply:
I understand your point. But there must be some logic in the application's behavior. If you don't know of this feature how would you guess it exists? 

if the same result can be achieved with `_before(test) ...` better to keep only one option here, so user won't be confused by some additional magical logic...

@abhimanyupandian  Also, is possible, could you update BDD chapter in docs to describe this use case there? 
Thanks!